package.json

```diff
@@ -12,8 +12,8 @@
   "author": "sebworks",
   "private": true,
   "scripts": {
-    "dev": "nuxt",
-    "build": "nuxt build",
+    "dev": "npm run i18n:get-translations && nuxt",
```

Comment:
Do we really need to get the translations every time we run dev?

Reply:
Definitely not.